src/main/java/co/zecko/retailer/client/httpClient/OrderClient.java

```diff
@@ -0,0 +1,76 @@
+package co.zecko.retailer.client.httpClient;
+
+import co.zecko.retailer.common.enums.HttpStatus;
+import co.zecko.retailer.common.pojo.collection.CollectionData;
+import co.zecko.retailer.common.pojo.order.OrderData;
+import co.zecko.retailer.common.pojo.order.OrdersData;
+import co.zecko.retailer.exception.BaseException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static co.zecko.retailer.common.constant.Header.AFTER_HEADER;
+import static co.zecko.retailer.common.constant.Header.BEFORE_HEADER;
+
+@FieldDefaults(level = AccessLevel.PACKAGE)
+public class OrderClient extends BaseClient {
+    static final String BASE_URI = "/retailer/v1/orders";
+    String zeckoAccessToken;
```

Comment:
Can we make final across all classes?

Reply:
@akartik80 we can't declare it as final, coz we are updating this zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);